README.md

```diff
@@ -6,9 +6,13 @@
 </a>
 
 # cosima-recipes
-Example recipes using the [`cosima-cookbook`](https://github.com/COSIMA/cosima-cookbook) infrastructure.
+This is a collection of example recipes and tutorials on analysing Ocean and SeaIce model output produced by [COSIMA: Consortium for Ocean-Sea Ice Modelling in Australia](http://cosima.org.au/) and its members. Most examples use output from the [ACCESS-OM2 model](https://doi.org/10.5194/gmd-13-401-2020), while some also use results from MOM6 configurations and remote sensing observations. To access the data used in these recipes you need an account and project with the Australian based [National Computational Infrastructure (NCI)](https://nci.org.au/)
```

Comment:
```suggestion
A collection of example recipes and tutorials on analysing ocean and sea ice model output produced by the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)](http://cosima.org.au/).

Most examples use output from the [ACCESS-OM2 model](https://doi.org/10.5194/gmd-13-401-2020), while some also use results from configurations of the Modular Ocean Model v6 (MOM6) and remote sensing observations.

To access the data used in these recipes you need an account with the Australian-based [National Computational Infrastructure (NCI)](https://nci.org.au/).
```

Reply:
Is MOM actually v6 ? Its just MOM6, and MOM6 will have its own versions?

Reply:
As far as I understand there is MOMX, X=1,2,3,4,5,6 but I doubt GFDL is doing any semantic versioning for MOM5 or MOM6.

But you are raising a fair point! My suggestion is to be verbose (Modular Ocean Model instead of MOM). And we can add a link to the MOM6 repo so people don't get confused whether this is v6 of MOM or some version of MOM6. So how about


```
[Modular Ocean Model 6 (MOM6)](https://github.com/mom-ocean/MOM6)
```

I don't even know what the main repository is tbh... [see https://forum.access-hive.org.au/t/mom6-code-where-it-lives-and-whats-the-best-way-to-know-what-version-im-using/118]